app/components/WidgetDynamicContent.tsx

```diff
@@ -220,6 +221,7 @@ export default class WidgetDynamicContent extends Component {
 
     attachEvents(container) {
```

Comment:
Minor:

Can we add a type annotation here?

```suggestion
    attachEvents(container: HTMLDivElement | null) {
```